src/lib/plugin_apis/fs.api

```diff
@@ -598,6 +599,76 @@ GType bd_fs_nilfs2_info_get_type () {
     return type;
 }
 
+#define BD_FS_TYPE_EXFAT_INFO (bd_fs_exfat_info_get_type ())
+GType bd_fs_exfat_info_get_type();
+
+/**
+ * BDFSExfatInfo:
+ * @label: label of the filesystem
+ * @uuid: uuid of the filesystem
+ * @sector_size: sector size used by the filesystem
+ * @cluster_size: cluster size used by the filesystem
+ * @sector_count: number of sectors in the filesystem
+ * @free_sectors: number of free sectors in the filesystem
+ * @cluster_count: number of clusters in the filesystem
+ * @free_clusters: number of free clusters in the filesystem
```

Comment:
Some mysterious disappearance of the struct members :zap:

Reply:
Yeah, I forgot to remove these from the docstring. Unfortunately the new exfatprogs don't provide these values.